options/options.go

```diff
@@ -14,6 +14,11 @@ type requestOptions struct {
 	SortBy     string
 }
 
+// getOptions implements options.
+func (r *requestOptions) getOptions() *requestOptions {
```

Comment:
type `requestOptions` must NOT implement `getOptions()` as it doesn't implement `options` interface on its own

Reply:
fixed